site/static/compare.html

```diff
@@ -43,7 +43,7 @@ <h3>Commits</h3>
     function add_datum_fields(datum) {
         let html = "";
         if (datum) {
-            html += "<td>" + datum.toFixed(2) + "</td>";
+            html += "<td>" + datum.toLocaleString('en-US', {minimumFractionDigits: 2, maximumFractionDigits: 2}) + "</td>";
```

Comment:
Is this supported across at least Chrome and Firefox?

Reply:
Yes. I checked that it works on current release versions of Firefox, Chrome, and Safari.